### PR TITLE
Add `flex-basis`

### PR DESCRIPTION
**Pull request by ellynliu**
Task: task-976841

Link: [preview](http://localhost:1111/)

Add `flex-basis` to Atlas

## Testing

Inspect and paste the following in the page (such as at the end of the article) to ensure that `flex-basis-0` works. I tried it out and the input boxes became the same size after applying `flex-basis-0`. If you remove `flex-basis-0`, then the boxes end up varying sizes. 

```
<div class="display-flex flex-direction-column flex-direction-row-tablet gap-xxs margin-bottom-sm">
	<div class="flex-grow-1 flex-basis-0">	
		<input class="input">
	</div>

	<div class="flex-grow-1 flex-basis-0">
		<div class="select">
			<select></select>
		</div>
	</div>
	
	<div class="flex-grow-1 flex-basis-0">	
		<input class="input">
	</div>
</div>
```

## Additional information

[Optional]

## Contributor checklist

- [x] Did you update the description of this pull request with a review link and test steps?
- [x] Did you submit a changeset? Changesets are required for all code changes.
- [ ] Does your pull request implement complex UI logic (js/ts)? Consider adding an integration test to test your user flow.
- [ ] Does your pull request add a new page to the documentation site? Add your new page for automated accessibility testing in /integration/tests/accessibility.

### CODE REVIEW

File: site/src/atomics/flex.md [hunk 1]
```diff
@@ -28,6 +29,7 @@ Flex atomics are useful when composing layouts and patterns containing other com
 | `align-content`   | `flex-start`, `flex-end`, `space-around`, `space-between`, `stretch`                | `tablet`   |
 | `flex-grow`       | `0`, `1`                                                                            | `tablet`   |
 | `flex-shrink`     | `0`, `1`                                                                            | `tablet`   |
+| `flex-basis`      | `0`, `1`                                                                            | `tablet`   |
```
Comment (ollips) [inline site/src/atomics/flex.md:32]: `auto` instead of `1`?

Comment (ellynliu) [reply site/src/atomics/flex.md]: changed

**Review by ollips — APPROVED** [review]

**Review by BenDMyers — APPROVED** [review]

File: css/src/atomics/flex.scss [hunk 2]
```diff
@@ -49,6 +49,10 @@ $flex-atomics: (
 	'flex-grow': (
 		0,
 		1
+	),
+	'flex-basis': (
+		0,
+		auto
```
Comment (wibjorn) [inline css/src/atomics/flex.scss:55]: Isn't `auto` the initial value?

Comment (ellynliu) [reply css/src/atomics/flex.scss:55]: it is...I guess I can remove it

**Pull request merged**